google/datalab/contrib/pipeline/airflow/setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+PROJECT=${1:-cloud-ml-dev}
+EMAIL=${2:-rajivpb@google.com}
+ZONE=${3:-us-central1}
+ENVIRONMENT=${3:-rajivpb-airflow}
```

Comment:
Should it be 4?

Reply:
please fix.

Reply:
Correct; fixed now.

Reply:
I think you fixed the composer one but not airflow one yet?

Reply:
Yeah, my bad - I kept thinking that all the comments referred to the composer's setup.sh and was a little confused on getting multiple comments for the same change. Will fix in a different PR mostly because I'm trying to get this in before demo time today.